kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerLocator.java

```diff
@@ -15,19 +15,37 @@
 
 package org.kie.server.services.impl;
 
+import org.kie.server.api.KieServerConstants;
+import org.kie.server.services.impl.storage.cloud.KieServerStateCloudRepository;
+
 public class KieServerLocator {
 
-    private static final KieServerImpl INSTANCE = new KieServerImpl();
-    
+    private static final KieServerImpl INSTANCE;
+    private static String strategy = System.getProperty(KieServerConstants.KIE_SERVER_STARTUP_STRATEGY);
```

Comment:
Can't you make this a local variable inside the static block?

Reply:
Committed suggested change

Reply:
We should be able to remove [line 24](https://github.com/kiegroup/droolsjbpm-integration/blob/c2aa7b08a764f40b65202ec8a997a24b8a112997/kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerLocator.java#L24) completely.

Reply:
Did you push this already? I still see it: https://github.com/rhtevan/droolsjbpm-integration/blob/kie-server-ocp-rearch-1/kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerLocator.java#L24